src/VisualStudio/IntegrationTest/TestUtilities/InProcess/QuickInfoToStringConverter.cs

```diff
@@ -28,6 +28,15 @@ private static string GetStringFromItem(object item)
                     return GetStringFromTextBlock(textBlock);
                 case ITextBuffer textBuffer:
                     return textBuffer.CurrentSnapshot.GetText();
+                case ContainerElement containerElement:
```

Comment:
📝 This was broken since #25181, but since all the Quick Info integration tests were disabled and never restored, no one ever knew.